genologics/descriptors.py

```diff
@@ -75,6 +76,10 @@ def __get__(self, instance, cls):
         instance.get()
         return instance.root.attrib[self.tag]
 
+    def __set__(self, instance, value):
+        instance.get()
+        instance.root.attrib[self.tag] = value
```

Comment:
Could you add test for the new setter and show it supports creation of new attribute if it does not exist?
test would be in test_descriptors.TestStringAttributeDescriptor
```python
    def test__set__(self):
        sd = self._make_desc(StringAttributeDescriptor, 'name')
        sd.__set__(self.instance, "test name2")
        assert self.et.attrib['name'] == "test name2"

    def test_create(self):
        instance_new = Mock(root=ElementTree.Element('test-entry'))
        bd = self._make_desc(StringAttributeDescriptor, 'name')
        bd.__set__(instance_new, "test name2")
        assert instance_new.root.attrib['name'] == "test name2"
```